### DISCUSSION TITLE
Montage EXRs

### DISCUSSION THREAD

**Discussion by dunnieboy** [ImageMagick/ImageMagick#7123] [Help]
### ImageMagick version

7.0.10-25 Q16 x64

### Operating system, version and so on

Windows 10 - Through Houdini PDG

### Description

I am using imagemagick to stich large tiles together to form a single massive exr.  Its working ..but its losing all the channels within the exr and the colorspace (ACES) is getting blown away.   Any ideas what command line arguments i need to keep all that data ?

Apologies if this has been covered a million times already.

**Comment by snibgo** [comment]
> 7.0.10-25 Q16 x64

Since that old version, IM has expanded its facilities for extra channels. I suggest you upgrade to the current version, and try again.
